Lib/fontTools/otlLib/builder.py

```diff
@@ -393,38 +395,38 @@ def buildFormat1Subtable(self, ruleset, chaining=True):
         st.Format = 1
         st.populateDefaults()
         coverage = set()
-        rulesets_by_first_glyph = {}
+        rulesetsByFirstGlyph = {}
         ruleAttr = self.ruleAttr_(format=1, chaining=chaining)
 
         for rule in ruleset.rules:
-            rule_table = self.newRule_(format=1, chaining=chaining)
+            ruleAsSubtable = self.newRule_(format=1, chaining=chaining)
```

Comment:
how about just ruleTable? ruleAsSubtable is a bit verbose

Reply:
It's not a table of rules.